cmd/nsm-init/nsm-init.go

```diff
@@ -46,6 +46,8 @@ const (
 	clientConnectionTimeout = time.Second * 60
 	// clientConnectionTimeout defines retry interval for establishing connection with the server
 	clientConnectionRetry = time.Second * 2
+	// location of network namespace for a process
+	netnsfile = "/proc/self/ns/net"
```

Comment:
I don't think this should be hard-coded.

Reply:
well it is kind of a constant in Linux for location of namespaces for a current process. Not sure what else I could do, do you have any suggestions?

Reply:
This is the unfortunate reality in Linux, things in /proc are simply hardcoded. I'm fine with leaving this here.